src/main/java/sg/ncl/MainController.java

```diff
@@ -953,6 +957,11 @@ public String editAccountDetails(
             errorsFound = true;
         }
 
+        if (errorsFound == false && !editUser.isPasswordValid()) {
+            redirectAttributes.addFlashAttribute("editPassword", "invalid");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "editPassword" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)